test/e2e/specs/all-features-confirmation/record/presentation.spec.js

```diff
@@ -201,6 +201,19 @@ describe('View existing record,', function() {
                 done.fail();
             });
         });
+        //Main Button
+        it('after clicking Main button, page should move up', function(done){
+            var mainButton = element(by.id('main-to-top')),
+                topOfPage = element(by.id('row-id'));
+
+            mainButton.click().then(function(){
+              expect(topOfPage.getLocation()).toEqual(jasmine.objectContaining({x: 30}, "Main button did not scroll up the page"));
```

Comment:
I know it's valuable to test this feature. But this way of testing seems wrong to me. It essentially is testing the margin of that element from top of the page which is css-based and can easily be changed. We shouldn't test the exact position of elements as it can easily be broken. 
And for testing this feature, it seems like there's no other way, so just remove the test cases. It should be one of the manual test steps.